cristin-commons/src/main/java/no/unit/nva/cristin/common/client/ApiClient.java

```diff
@@ -87,6 +97,7 @@ public ApiClient(HttpClient client) {
     }
 
     public static <T> T fromJson(String body, Class<T> classOfT) throws IOException {
+        logger.info("json fra cristin: {}", body);

```

Comment:
Is this needed for get requests? will it fill up a lot of log space?

Reply:
Not needed at all... manual debug...